### PR TITLE
EAT-UCCS glue CDDL

### PR DESCRIPTION
**Pull request by thomas-fossati**

### CODE REVIEW

File: draft-ietf-rats-uccs.md [hunk 1]
```diff
@@ -456,6 +456,15 @@ CWT-kid = bytes
 ~~~
 {: sourcecode-name="uccs-additional-examples.cddl"}
 
+Specifications that need to use UCCS in the context of EAT {{-eat}} need the glue CDDL defined in {{fig-uccs-eat-glue}}.
```
Comment (thomas-fossati) [inline draft-ietf-rats-uccs.md]:
```suggestion
Specifications that use UCCS in the context of EAT {{-eat}} need the glue CDDL defined in {{fig-uccs-eat-glue}}.
```

Comment (henkbirkholz) [reply draft-ietf-rats-uccs.md]: okay

**Review by henkbirkholz — APPROVED** [review]
okay with nitfix

**Review by quic-jodonogh — APPROVED** [review]

**Comment by carl-wallace** [comment]
Should the last definition be
```
$EAT-CBOR-Tagged-Token /= UCCS
```

File: draft-ietf-rats-uccs.md [hunk 2]
```diff
@@ -456,6 +456,15 @@ CWT-kid = bytes
 ~~~
 {: sourcecode-name="uccs-additional-examples.cddl"}
 
+Specifications that use UCCS in the context of EAT {{-eat}} need the glue CDDL defined in {{fig-uccs-eat-glue}}.
+
+~~~ cddl
+$JSON-Selector-Type /= "UCCS"
+$JSON-Selector-Value /= ~UCCS
+$EAT-CBOR-Tagged-Token = UCCS
```
Comment (henkbirkholz) [inline draft-ietf-rats-uccs.md]:
```suggestion
$EAT-CBOR-Tagged-Token /= UCCS
```

thanks to @carl-wallace for  https://github.com/ietf-rats-wg/draft-ietf-rats-uccs/pull/17#issuecomment-1712129351

**Comment by henkbirkholz** [comment]
yes, type-choice seems useful in a type-choice socket :relaxed:

**Comment by thomas-fossati** [comment]
I think this can be retired.  My understanding is that @cabo and @laurencelundblade are taking care of this bit.

**Pull request closed without merging**